phoenix-core/src/test/java/org/apache/phoenix/compile/QueryCompilerTest.java

```diff
@@ -6981,4 +6981,31 @@ public void testReverseVarLengthRange6916() throws Exception {
             assertEquals(explainExpected, explainPlan);
         }
     }
+
+    @Test
+    public void testUncoveredPhoenix6969() throws Exception {
+
+        try (Connection conn = DriverManager.getConnection(getUrl());
+                Statement stmt = conn.createStatement();) {
+
+            stmt.execute(
+                "create table dd (k1 integer not null, k2 integer not null, k3 integer not null,"
```

Comment:
table name to be `generateUniqueName()`?

Reply:
Not needed for these unit tests, there is no state preserved between connections.